.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and Publish JAR Packages
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish pre-release' || format('Release {0}', inputs.release_type)}} Package by @${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: The type of release
+        options:
+        - Major
+        - Minor
+        - Patch
+        - Snapshot
+      publish_vulnerabilities:
+        type: string
+        default: true
+      dotnet_version:
+        type: string
+        default: 5.0
+
+jobs:
+  build-and-pubish:
+    name: Build and publish JAR packages to Maven repository
+    uses: IABTechLab/uid2-shared-actions/.github/workflows/shared-publish-to-nuget-versioned.yaml@kcc-UID2-2329-add-dotnet-pipeline
```

Comment:
Will revert this after https://github.com/IABTechLab/uid2-shared-actions/pull/79 is merged